lib/Dialects/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -137,11 +137,11 @@ Operation *ReverseOp::getTiledImplementation(OpBuilder &builder,
   SmallVector<Type, 4> resultTypes;
   if (hasTensorSemantics()) {
     tiledOperands.emplace_back(
-        getSlice(builder, loc, output(), mirrorOffsets, sizes, strides));
+        getSlice(builder, loc, outputs[0], mirrorOffsets, sizes, strides));
```

Comment:
Why this change ?
It's not like the algorithm as written supports multi-output, so I'd expect either `output()` or an early exit somewhere that there is a single `output` in this instance of the op (i.e. `outputs.empty()`).

Reply:
`output()` would refer to the destination of the untiled operation. I need to rework this a bit to be the `iter_args` of the inner loop. This is WIP, so technically it is correct, but does not work for reductions. Hoping to land this clean up before changing it to handle reduction cases as well.